presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1418,6 +1418,24 @@ public void testGroupingCombinationsDistinct()
     }
 
     @Test
+    public void testIntersect()
+            throws Exception
+    {
+        MaterializedResult actual = computeActual("SELECT * FROM (VALUES 1, 2, 3, 4) INTERSECT SELECT * FROM (VALUES 3, 4) INTERSECT SELECT * FROM (VALUES 4)");
+        MaterializedResult expected = resultBuilder(getSession(), BIGINT, BIGINT)
+                .row(4)
+                .build();
+
+        assertEquals(actual.getMaterializedRows(), expected.getMaterializedRows());
```

Comment:
does the other database not support `INTERSECT`?

you can rewrite this as:
`
assertEquals(query, "values 4");